### PR TITLE
ITKdev - CMN-577 - favorites list material component

### PR DESCRIPTION
**Pull request by sinejespersen**
#### Link to issue

https://jira.itkdev.dk/browse/CMN-608

#### Description

Added a component
removed unused file
renamed recommender-material to simple-material and moved the files to components (https://github.com/danskernesdigitalebibliotek/dpl-design-system/pull/220)
created an adapter to simple-material for favorites-list
renamed card-list to CardListItemAdapter 
added some GuardedAppId's

#### Screenshot of the result

image

#### Checklist

- [ ] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [ ] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [ ] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

#### Additional comments or questions

If you have any further comments or questions for the reviewer them please add them here.

### CODE REVIEW

**Review by yepzdk — APPROVED** [review]

File: src/apps/recommender/InspirationRecommender.tsx [hunk 1]
```diff
@@ -41,7 +41,10 @@ const InspirationRecommender: FC = () => {
       <ul className="recommender__grid">
         {recommendedMaterials &&
           recommendedMaterials.search.works.map((work) => (
-            <RecommendMaterial work={work as Work} />
+            <RecommendMaterial
+              appTag="inspiration-recommender"
```
Comment (spaceo) [inline src/apps/recommender/InspirationRecommender.tsx:45]: What is the purpose of `appTag`? It is the type of a `GuardedAppId` but it is not, as far as I can see, using the guardedApp functionality.

**Review by spaceo — COMMENTED** [review]
Need to have clarifed some questions :)

File: src/apps/favorites-list/FavoritesList.tsx [hunk 2]
```diff
@@ -48,7 +48,7 @@ const FavoritesList: React.FC<FavoritesListProps> = ({ pageSize }) => {
         <ul className="card-list-page__list my-32">
           {displayedMaterials.map((pid) => (
             <li key={pid}>
-              <CardList key={pid} pid={pid} />
+              <CardListItemAdapter key={pid} pid={pid} />
```
[the remark this answers is not in the document]
Comment (sinejespersen) [reply src/apps/favorites-list/FavoritesList.tsx:51]: I renamed a card-list to card-list-item-adapter, as this was what it actually was 

https://github.com/danskernesdigitalebibliotek/dpl-react/pull/440/files#diff-188dbd819dbbc4db9500f1e9e48d853d235b53df02d5116a9c7fe174577b3a4aR5-R11

File: src/apps/recommender/InspirationRecommender.tsx (hunk 1 above)
Comment (sinejespersen) [reply src/apps/recommender/InspirationRecommender.tsx:45]: https://github.com/danskernesdigitalebibliotek/dpl-react/pull/440/files#diff-a1cd3c7df4bbe6dec8c1f3bb698cb32021057973376864601e5bffb80e630371R59-R62

It ends up in the simple-material, which creates a guarded request

**Review by spaceo — COMMENTED** [review]
Can we talk about this in an online meeting briefly? There are some things I do not understand. Think it will be a faster review process that way.

File: src/apps/recommender/InspirationRecommender.tsx (hunk 1 above)
Comment (sinejespersen) [reply src/apps/recommender/InspirationRecommender.tsx]: renamed

**Review by spaceo — APPROVED** [review]
LGTM

**Pull request merged**